src/core/iomgr/iomgr.h

```diff
@@ -37,11 +37,22 @@
 /* gRPC Callback definition */
 typedef void (*grpc_iomgr_cb_func)(void *arg, int success);
 
+typedef struct grpc_iomgr_cb {
+  grpc_iomgr_cb_func cb;
+  void *cb_arg;
+  int success;
+  int is_ext_managed;  /** is memory being managed externally? */
+  struct grpc_iomgr_cb *next;  /** Do not touch */
+} grpc_iomgr_cb;
+
+grpc_iomgr_cb *grpc_iomgr_cb_create(grpc_iomgr_cb_func cb, void *cb_arg,
```

Comment:
We ultimately won't need this routine.

However, consider providing a grpc_iomgr_closure_init(grpc_iomgr_closure *, ...) to save line count across the project.